events/wc_dark_cult_events.txt

```diff
@@ -1086,9 +1145,336 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
+	desc = {
+		trigger = { NOT = { has_character_flag = hunted_by_garona } }
+		text = EVTDESC_WCDAC_5041_first_kill
+	}
+	desc = {
+		trigger = {
+			trigger = { has_character_flag = hunted_by_garona }
+		}
+		text = EVTDESC_WCDAC_5041_later_kills
+	}
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	trigger = {
+		c_10620 = {
+			is_alive = yes
+			is_dying = no
+	
+			is_untouchable_trigger = no
+	
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			is_inaccessible_trigger = no
+			
+			is_landed = no
+			
+			NOT = { religion = old_gods_worship }
+			NOT = { is_married = 52000 }
+			NOT = { obedient = 52000 }
+		}
+		character = 52000
+	}
+	
+	fail_trigger_effect = {
+		# Skip event
+		character_event = { id = WCDAC.5045 months = 12 }
+		clr_character_flag = hunted_by_garona
+	}
+	
+	immediate = {
+		random_courtier = {
+			limit = {
+				NOT = { character = 10620 }
+				religion = old_gods_worship
+				is_adult = yes
+				NOR = {
+					trait = in_hiding
+					trait = travelling
+				}
+				is_incapable = no
+				combat_rating_diff = { who = 10620 value < 10 }
+			}
+			death = { death_reason = death_murder }
+		}
+		c_10620 = {
+			save_event_target_as = target_garona
+			add_trait = travelling
```

Comment:
So, I married Garona after she gained the trait. After 2 years, the trait isn't cleared.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86472378-1ecce800-bd50-11ea-9180-9bcfdd219d08.png)
![image](https://user-images.githubusercontent.com/46384992/86472518-53d93a80-bd50-11ea-8deb-8b66641a8819.png)

</details>